backend/ipmi.pm

```diff
@@ -76,18 +76,18 @@ sub restart_host {
 
     $self->ipmitool("chassis power off");
     while (1) {
+        sleep(3);
```

Comment:
I know you are just moving a sleep but could you please add a comment why it's necessary?